src/components/ui/file-upload.tsx

```
@@ -0,0 +1,252 @@
+      URL.revokeObjectURL(photoToRemove.url);
+    }
+    onPhotosChange(photos.filter((photo) => photo.id !== photoId));
+  };
+
+  const handleOpenFilePicker = () => {
+    fileInputRef.current?.click();
+  };
+
+  React.useEffect(() => {
+    return () => {
+      photos.forEach((photo) => URL.revokeObjectURL(photo.url));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)